pages/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/publishing_checklist.md

```diff
@@ -9,41 +9,49 @@ contributors:
   - sebastianbenz
 ---
 
-Follow this checklist to ensure your pages are getting all the benefits AMP has to offer by being healthy and accessible on the web! 
-
+Follow this checklist to give your site the fullest AMP experience! 
 
 # Ensure AMP Specification Validation 
 
-AMP comes with a ton of built in benefits, such as decreasing content load times by preloading content from AMP Caches. However, only documents that can be verified as valid AMP get these benefits. If you publish documents that have critical AMP errors your pages will not be indexed by an AMP cache, or possible served as error pages. 
+AMP comes with a ton of built in benefits, such as decreasing user wait time by preloading content from AMP Caches. To get these benefits, pages must be valid AMP documents. Pages published with critical AMP errors are not indexable by AMP Caches, and possibly served as error pages.
 
-Learn more about the tools you can use to validate your AMP pages:
+Never publish an invalid AMP page again by using these tools:
 
+* [Validate AMP pages](../../../documentation/guides-and-tutorials/learn/validation-workflow/validate_amp.md?format=websites)
+* [The AMP Validator ](https://validator.ampproject.org/)
+* [Google AMP Tester](https://search.google.com/test/amp)
+* [AMP Tools](../../../documentation/tools.html?format=websites) 
 
 
-*   [Validate AMP pages](../../../documentation/guides-and-tutorials/learn/validation-workflow/validate_amp.md?format=websites)
-*   [The AMP Validator ](https://validator.ampproject.org/)
-*   [Google AMP Tester](https://search.google.com/test/amp)
-*   [AMP Tools](../../../documentation/tools.html?format=websites) 
+# Grant cached AMP pages server access
 
+Great news, valid AMP pages automatically opt into all existing AMP Caches! This means your users experience content that loads efficiently and safely. These types of optimizations are great, but come with a small catch.
+Some users will be served AMP pages from domains that do not match your own. This can cause pages to lose access to site data when using dynamic AMP components such as [`<amp-form>`](../../../documentation/components/reference/amp-form.md?format=websites) or [`<amp-list>`](../../../documentation/components/reference/amp-list.md?format=websites). These types of errors are Cross-Origin Resource Sharing, or CORS, issues. Work with safety, not against it, by enabling CORS Requests from all available [AMP Caches](https://cdn.ampproject.org/caches.json)! 
```

Comment:
Add: `If you're using Node.js in your backend, you can use the [amp-cors middleware](https://github.com/ampproject/amp-toolbox/tree/master/packages/cors).`